wolfCLU/clu_src/tools/clu_funcs.c

```diff
@@ -340,6 +344,24 @@ void wolfCLU_certHelp()
 }
 
 void wolfCLU_genKeyHelp() {
+
+        const char* keysother[] = { /* list of acceptable key types */
+        "KEYS: "
+    #ifdef HAVE_ED25519
+        ,"ed25519"
+    #endif
+    #ifdef HAVE_ECC
+        ,"ecc"
+    #endif
+    #ifdef HAVE_CURVE25519
```

Comment:
Curve 25519 is just that, a curve, sometimes referred to as the Edwards-form Curve25519 but not a key generation option, let's remove this from the help printout in key generation schemes.